docs/essentials/collections/threadsafe/blockingCollectionOverview.md

```diff
@@ -1 +1,115 @@
+# BlockingCollection Overview
+
+[BlockingCollection&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.BlockingCollection%601.html) is a thread-safe collection class that provides the following features:
+
+*   An implementation of the Producer-Consumer pattern.
+
+*   Concurrent adding and taking of items from multiple threads.
```

Comment:
Hmmm.  I think it's tricky to describe that value prop without describing the producer-consumer problem. Perhaps something like "A way to have thread-safe addition and removal of data from a collection"?  It doesn't quite capture the purpose of using this, but I think that is a value prop that the original bullet point was trying to convey.